.circleci/config.yml

```diff
@@ -188,12 +182,11 @@ jobs:
               then
                 git clone https://github.com/facebookresearch/habitat-sim.git --recursive
               fi
-              # while [ ! -f ./cuda_installed ]; do sleep 2; done # wait for CUDA
               export PATH=$HOME/miniconda/bin:/usr/local/cuda/bin:$PATH
               . activate habitat;
               cd habitat-sim
               pip install -r requirements.txt --progress-bar off
-              pip install pillow
+              pip install imageio imageio-ffmpeg
```

Comment:
Why do we need those? Should they be part of the requirements of they are needed ?

Reply:
Ok, can you merge it now?